assets/onboarding/index.jsx

```diff
@@ -1,24 +1,28 @@
-import { Stepper } from '@woocommerce/components';
 import { render } from '@wordpress/element';
-import { useFullScreen } from './use-fullscreen';
+import { useFullScreen } from '../react-hooks';
 import { steps } from './steps.jsx';
+import { QueryStringRouter } from './query-string-router';
+import Navigation from './navigation';
+import ContentContainer from './content-container';
 
-function SenseiOnboardingPage() {
-	useFullScreen( [ 'sensei-color', 'sensei-onboarding__page' ] );
+/**
+ * Query string name used to route the onboarding wizard.
+ */
+const QUERY_STRING_NAME = 'onboarding-step';
```

Comment:
This can probably be just `step` to keep the url a bit easier on the eye :)

Reply:
Makes sense!
I added a different name to prevent some conflict if we have different navigation at the same time. But I think it'll not happen in this case. So I updated to `step` that is really better! =)